index.md

```diff
@@ -1,152 +0,0 @@
----
```

Comment:
Are you sure about removing this?

Isn't this the homepage of the new tutorial?

Reply:
oops....not sure how this got in the commit.